src/core/document.js

```diff
@@ -855,6 +856,13 @@ class PDFDocument {
     return this.xfaFactory !== null;
   }
 
+  get htmlForXfa() {
+    if (this.isPureXfa) {
```

Comment:
Is the `isPureXfa` getter still necessary now, since just `if (this.xfaFactory) {` ought to suffice (which is also consistent with other parts of this file)?